<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputPortConfigWidget</class>
 <widget class="QWidget" name="OutputPortConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="portNumLBL">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeCB"/>
   </item>
   <item>
    <widget class="QLineEdit" name="colorF">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minValF">
     <property name="text">
      <string>Min Val</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="midValF">
     <property name="text">
      <string>Mid Val</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxValF">
     <property name="text">
      <string>Max Val</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reverseBox">
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeCB</tabstop>
  <tabstop>colorF</tabstop>
  <tabstop>minValF</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
